### DISCUSSION TITLE
Share variable from C# to Powershell

### DISCUSSION THREAD

**Discussion by mika76** [dotnet/interactive#1984] [Q&A]
I was trying to use powershell to `Set-Clipboard -Value $outputvar` and I even used `#!share --from csharp outputvar` but that did not seem to work. Should this be working, or is my syntax wrong?

**Comment by jonsequitur** [comment]
I tried this just now and it worked for me. Could you open an issue with some additional details?

**Reply by mika76** [reply]
Logged as #1996 - I forgot to mention here I'm on MacOS
